redditAPI.py

```diff
@@ -74,6 +85,7 @@ def get_posts(query, cap=None):
         for post in posts:
             title = post['data'].get('title')
             if title is not None and title not in titles:
+                title = clean_title(title)
```

Comment:
Don't send the cleaned titles to display, save them in a different array that can be used to do ML. Display original titles on the search page.

Reply:
Done!